program/tests/stake_instruction.rs

```diff
@@ -1037,6 +1037,7 @@ fn test_stake_checked_instructions() {
 }
 
 #[test]
+#[allow(deprecated)] // TODO: remove need for this
```

Comment:
Can you double check these are necessary? I commented them out, ran tests, and didn't see any warnings. If they are, can you add more to the comment on what is tripping it up for the future dev?